src/renderer/equalizerApi.ts

```diff
@@ -80,21 +82,23 @@ export const getMainPreAmp = (): Promise<number> => {
   window.electron.ipcRenderer.sendMessage('peace', [channel, 5, 5, 0]);
 
   const responseHandler = buildResponseHandler<number>((result, resolve) => {
-    const OVERFLOW_OFFSET = 4294967296;
+    const MAX_GAIN = 30;
+    const MIN_GAIN = -30;
 
     // If gain is larger than 30, assume overflow occured.
```

Comment:
Add a comment to explain this overflow thing better. In particular, it seems like we are getting an unsigned number back from peace so negative gains are really large.